src/ui/variables.scss

```diff
@@ -0,0 +1,24 @@
+// COLORS
+// -- Primary Colors --
+$whites-1: rgb(255, 255, 255);
+$whites-2: rgb(250, 250, 250);
+$blacks-1: rgb(0, 0, 0);
+$blacks-2: rgb(51, 51, 51);
+
+// -- Alpha Colors --
+$alpha-black-1: rgba(0, 0, 0, 0.15);
+
+// -- Common Colors --
+// Background colors
+$background-color-white: $whites-2;
+
+// Font Colors
+$font-color-1: $blacks-2;
+$font-color-2: $whites-1;
+
+// FONTS
+// -- Font Sizing --
+$font-size-1: 1rem;
+$font-size-2: 1.2rem;
+$font-size-3: 1.6rem;
+$font-size-4: 2rem;
```

Comment:
is it standard practice to put all these in one big file? Would we keep it as is as it grows bigger and bigger?

Reply:
definitely not standard, but since we don't have that many right now I felt like it was unnecessary for now. we could and probably should in the future to something like `colors.scss`, `fonts.scss`, etc